exercises/hamming/test/hamming_test.dart

```diff
@@ -1,84 +1,57 @@
-import 'package:test/test.dart';
 import 'package:hamming/hamming.dart';
+import 'package:test/test.dart';
+
+final hamming = Hamming();
 
 /// Test cases match one-to-one with those described
 /// in the problem specification found here:
 /// https://github.com/exercism/problem-specifications/blob/master/exercises/hamming/canonical-data.json
 void main() {
-  final Hamming hamming = new Hamming();
+  final mustEqual = predicate((ArgumentError e) => e.message == 'left and right strands must be of equal length',
```

Comment:
Did you forget this one or is this intentional?

Reply:
About your comment on analyzer confusing the user. Either of the following should be sufficient
```dart
predicate((Object e) => e is ArgumentError && e.message == '...', '...');
```
```dart
predicate<dynamic>((dynamic e) => e is ArgumentError && e.message == '...', '...');
```

Reply:
Hmm, I thought I did that....

Reply:
Ah... I didn't commit the change. Pushing it out now